src/sx-design/src/Tabs/Tabs.js

```diff
@@ -17,30 +17,96 @@ type Props = {
  *
- * TODO: improve accessibility
+ * ## Accessibility
+ *
+ * When focus moves into the tab list, places focus on the active tab element. When the tab list
+ * contains the focus, moves focus to the next element in the page tab sequence outside the tablist,
+ * which is typically either the first focusable element inside the tab panel or the tab panel itself.
+ *
+ * When focus is on a tab element in a horizontal tab list:
+ *  - Left Arrow: moves focus to the previous tab. If focus is on the first tab, moves focus to the
+ *   last tab.
+ *  - Right Arrow: moves focus to the next tab. If focus is on the last tab element, moves focus to
+ *   the first tab.
+ *
+ * When focus is on a tab in a tablist:
+ *  - Space or Enter: activates the tab if it was not activated automatically on focus.
+ *
+ * For more information about accessibility please visit:
  *  - https://www.w3.org/TR/wai-aria-practices/#tabpanel
  *  - https://www.w3.org/TR/wai-aria-practices/examples/tabs/tabs-2/tabs.html
  */
 export default function Tabs(props: Props): Node {
+  const tabRefs = useRef([]);
+
+  // The following effect takes care of switching the tabs based on the pressed arrows (left/right).
+  // We could emulate this behavior by simply using radiogroup which does this by default. However,
+  // it's convenient to implement it manually because we want to delay the tabpanel switching on
+  // spacebar/enter press. This way we can preload the tabpanel content in case the panel is not
+  // loading instantly (for example: https://relay.dev/docs/api-reference/use-preloaded-query/).
+  //
+  // TODO: add ability to activate the tabs instantly when the preloading is not needed (?)
+  useEffect(() => {
+    const keys = {
+      left: 37,
+      right: 39,
+    };
+
+    const direction = {
+      [37]: -1, // left
+      [39]: 1, // right
+    };
+
+    const keypressHandler = (event) => {
+      const activeIndex = tabRefs.current.findIndex((tab) => tab === document.activeElement);
+
+      if (activeIndex === -1) {
+        // none of the tabs is focused so we have to ignore all attempts to change the tab with arrows
+        return;
+      }
+
+      const keyCode = event.keyCode;
+      if (keyCode === keys.left || keyCode === keys.right) {
+        const nextTab = tabRefs.current[activeIndex + direction[keyCode]];
+        if (nextTab != null) {
+          nextTab.focus();
+        } else if (keyCode === keys.left) {
+          // focus LAST tab (going from the left card to the end)
+          tabRefs.current[tabRefs.current.length - 1]?.focus();
+        } else if (keyCode === keys.right) {
+          // focus FIST tab (going from the right card to the beginning)
+          tabRefs.current[0]?.focus();
+        }
+      }
+    };
+
+    window.addEventListener('keydown', keypressHandler);
+    return () => {
+      window.removeEventListener('keydown', keypressHandler);
+    };
```

Comment:
TODO: it would be nice to reuse `useKeyPress.js` from https://github.com/adeira/universe/pull/3076